lib/screens/profile_screens/contribution/contribution_detail/components/contribution_detail_subtitle.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+
+class ContributionDetailSubtitle extends StatelessWidget {
+  final String title;
+
+  const ContributionDetailSubtitle({
+    Key? key,
+    required this.title,
+  }) : super(key: key);
```

Comment:
`  const ContributionDetailSubtitle(this.title, {Key? key}) : super(key: key);`

Reply:
changed 👍